src/controllers/user.controller.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const userService = require('../services/user.service');
+
+const getAll = async(req, res) => {
+  res.send(await userService.readAll());
+};
+
+const get = async(req, res) => {
+  const { id } = req.params;
+  const user = await userService.read(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.status(200).json(user);
+};
+
+const create = async(req, res) => {
+  const body = req.body;
```

Comment:
Consider using object destructuring to access the name property of req.body. This would make your code more concise and easier to read.